explorer/client/src/components/transaction-card/RecentTxCard.tsx

```diff
@@ -112,7 +112,9 @@ function LatestTxView({
     const totalCount = results.totalTxcount || 1;
     const [searchParams, setSearchParams] = useSearchParams();
     const pageParam = parseInt(searchParams.get('p') || '1', 10);
-    const [showNextPage, setShowNextPage] = useState(true);
+    const [showNextPage, setShowNextPage] = useState(
+        Math.ceil(NUMBER_OF_TX_PER_PAGE * 1) < totalCount
```

Comment:
```suggestion
        NUMBER_OF_TX_PER_PAGE < totalCount
```

nit: `Math.cei` and `*1` seem to be redundant

Reply:
You are right 😄